doc/nrf/index.rst

```diff
@@ -3,6 +3,13 @@
 Welcome to the |NCS|!
 #####################
 
+.. important::
+
+   |NCS| v1.4.99-dev1 is a development release and will be replaced by v1.5.0 in the future.
```

Comment:
I think the term `development tag` is more correct, and will give the reader a more correct interpretation of the tag, compared to a release.

```suggestion
   |NCS| v1.4.99-dev1 is a development tag and will be replaced by v1.5.0 in the future.
```